stories/NavBar.stories.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+
+import { NavBar, Link, Button } from '../src';
+import { color } from '../src/styles';
+
+storiesOf('NavBar', module)
+  .addParameters({
+    component: NavBar,
+  })
+  .addWithJSX(
+    'default',
```

Comment:
I think you can get rid of the `default` and `default with logo` cases here - they are demonstrations of something we likely will never do.  At the very least, reorder and rename the stories so that the default is the navbar shown on thegymnasium.com

Reply:
@mbifulco 
I believe you are referring to the *secondary navigation* for thegymnasium.com, which is a special case preconfigured instance of the NavBar.

These stories demonstrate the capabilities of this component, and I recommend we keep them for future reference. 

There will be a separate story for the `SecondaryNavigation` that will be exactly what you're looking for.

If you still want me to change these stories, please let me know.